sammba/registration/struct.py

```diff
@@ -617,6 +625,11 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
         out_tstat_warp_head = tstat(in_file=out_tcat.outputs.out_file,
                                     outputtype='NIFTI_GZ')
 
+        nwarp_adjusted_mean = 'warped_{0}_adjusted_mean.nii.gz'.format(n_iter)
```

Comment:
have you checked if this file is created in the `write_dir` ?

Reply:
yep it is